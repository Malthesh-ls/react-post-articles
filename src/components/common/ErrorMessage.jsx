import React from 'react';

function ErrorMessage() {
  return <div>Error occurred, Please try again!</div>;
}

export default ErrorMessage;
