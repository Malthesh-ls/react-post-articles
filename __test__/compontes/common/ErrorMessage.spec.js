import renderer from 'react-test-renderer';
import ErrorMessage from '../../../src/components/common/ErrorMessage';

describe('ErrorMessage Tests', () => {
  it('should match snapshot', () => {
    const snapshot = renderer
      .create(<ErrorMessage />)
      .toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
