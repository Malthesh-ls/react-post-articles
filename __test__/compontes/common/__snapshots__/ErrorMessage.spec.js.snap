// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage Tests should match snapshot 1`] = `
<div>
  Error occurred, Please try again!
</div>
`;
